### DISCUSSION TITLE
The same relationship

### DISCUSSION THREAD

**Discussion by telyakov** [structurizr/lite#126] [Q&A]
### Description

The same relationship between a container and a component in another container is not displayed and no error is generated.

### Steps to reproduce

eraProduction.acl_frontend.store -> eraProduction.acl_bff.gas_separated_research_protocol_controller "Visible" ${ENVOY_SERVICE_CALL} {
    perspectives {
        ${USECASES_ACL_RESEARCH_PROTOCOL_VIEW} ${USECASES_ACL_RESEARCH_PROTOCOL_VIEW_DESCRIPTION}
    }
}

eraProduction.acl_frontend.store -> eraProduction.acl_bff.gas_separated_research_protocol_controller "Hidden" ${ENVOY_SERVICE_CALL} {
    perspectives {
        ${USECASES_ACL_RESEARCH_PROTOCOL_VIEW} ${USECASES_ACL_RESEARCH_PROTOCOL_VIEW_DESCRIPTION}
    }
}

### Severity

Minor

### Priority

I have no budget and there's no rush, please fix this for free

**Comment by simonbrowndotje** [comment]
I imagine you're seeing the implied relationships feature -> https://docs.structurizr.com/dsl/cookbook/implied-relationships/

(if this is not the case, please provide a full `workspace.dsl` file)

**Comment by telyakov** [comment]
I install latest version (2024.03.03), all relations showed, sorry...
